client/components/loading.jsx

```diff
@@ -0,0 +1,13 @@
+import React from 'react'
+
+const Loading = () => {
+  return (
+    <div className="ui active dimmer">
+      <div className="ui big text loader">
```

Comment:
You can also use:
```js
<Loader active />
```
see docs: https://react.semantic-ui.com/elements/loader/#states-active